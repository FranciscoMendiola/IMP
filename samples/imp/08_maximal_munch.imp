iff := 10
whilex := 20
x123 := 456
abc := xyz
