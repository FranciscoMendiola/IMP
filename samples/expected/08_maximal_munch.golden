Token {tokenKind = Id, lexeme = "iff", position = 0}
Token {tokenKind = Assign, lexeme = ":=", position = 4}
Token {tokenKind = IntLit, lexeme = "10", position = 7}
Token {tokenKind = Id, lexeme = "whilex", position = 10}
Token {tokenKind = Assign, lexeme = ":=", position = 17}
Token {tokenKind = IntLit, lexeme = "20", position = 20}
Token {tokenKind = Id, lexeme = "x123", position = 23}
Token {tokenKind = Assign, lexeme = ":=", position = 28}
Token {tokenKind = IntLit, lexeme = "456", position = 31}
Token {tokenKind = Id, lexeme = "abc", position = 35}
Token {tokenKind = Assign, lexeme = ":=", position = 39}
Token {tokenKind = Id, lexeme = "xyz", position = 42}
